### PR TITLE
Created new function prepareCreateSubSpaceExtrinsic()

### LINKED ISSUE #200: [C4GT] ChainSpace: Separate preparation of create subspace extrinsic from chain dispatch
## Description
Separate chain-space creation preparation and dispatch of extrinsic to chain. This would help when bulk transaction takes place so we just prepare all the required extrinsics and then execute these transaction in batches by dumping them into a promise using 
a batch sign & submit utility ex, `authorizeBatch` which can take array of extrinsics, instead of just one.

## Goals

- [ ] Creation of new export function named `prepareCreateSubSpaceExtrinsic()`
- [ ] Integration of above method into existing `dispatchSubspaceCreateToChain()`

## Implementation Details
Developers/Contributors can refer to example PRs: #188, #182

---- 

## Product Name
CORD.JS

## Organization Name
Dhiway

## Domain
Blockchain, SDK Development, API

## Tech Skills Needed
Typescript 

## Mentor(s)
@amarts @NiranjanAP @vatsa287 @adi-a11y 

## Complexity
[Low]

## Category
[Performance Improvement], [Feature]

## Sub Category
 [API], [Refactoring], [Backend], [Beginner friendly]

### PR DESCRIPTION
**Pull request by kartikaysaxena**
Fixes #200

### CODE REVIEW

**Comment by kartikaysaxena** [comment]
PTAL @vatsa287  @adi-a11y

File: packages/chain-space/src/ChainSpace.chain.ts [hunk 1]
```diff
@@ -308,6 +309,42 @@ export async function dispatchSubspaceCreateToChain(
...
+ * @param chainSpace - The ChainSpace object containing necessary information for creating the ChainSpace on the blockchain.
+ * @param authorAccount - The blockchain account used for signing and submitting the transaction.
+ * @param count - The count of subspaces to be created.
+ * @param creatorUri - The DID URI of the creator, used to authorize the transaction.
+ * @returns The prepared extrinsic ready for batch signing and submitting.
+ */
+export async function prepareCreateSubSpaceExtrinsic(
+  chainSpace: IChainSpace,
+  authorAccount: CordKeyringPair,
+  count: number,
```
Comment (vatsa287) [inline packages/chain-space/src/ChainSpace.chain.ts]: `count` here refers to count of transactions permitted to be performed on the chain for the subspace.

**Review by vatsa287 — CHANGES_REQUESTED** [review]
Please declare `prepare...` function above `dispatch....` function.

File: packages/chain-space/src/ChainSpace.chain.ts (hunk 1 above)
Comment (kartikaysaxena) [reply packages/chain-space/src/ChainSpace.chain.ts]: Updated, thanks for the review!

**Review by vatsa287 — APPROVED** [review]
LGTM

**Pull request merged**